src/http/client.rs

```diff
@@ -1765,6 +1765,26 @@ impl Http {
         .await
     }
 
+    /// Follow an Announcement Channel to send messages to a target channel.
+    pub async fn follow_news_channel(
```

Comment:
Should this be named `follow_news_channel` or `follow_announcement_channel`?

Reply:
The API calls them news channels, but I saw existing references to announcement channels in serenity. I thought for the raw http module I would follow the api name then could just call it `follow` and avoid the naming bikeshedding.

Reply:
Maybe we should rename the existing references to news channels?

Reply:
Should this been done in a different PR?

Reply:
We should strive to be consistent as possible, so yes, a pull request targeting `next` should be opened to replace instances of "announcement channel" with "news channel".

Reply:
Why not just target this PR at next and replace all references in this PR?

Reply:
Probably best to keep this PR like it is and maybe make a follow up PR

Small PRs are good anyways